projects/project-player-library/src/lib/assets/i18n/en.json

```diff
@@ -45,7 +45,7 @@
     "NO_FILES":"No {{fileType}} found for this project",
     "DELETE_ATTACHMENT":"Are you sure you want to delete this attachment ?",
     "REMOVE":"Remove",
-    "CERTIFICATE":"CERTIFICATE",
+    "CERTIFICATE":"Certificate",
```

Comment:
Why this change?

Reply:
first it was capital case as we have camalize case for the header component.